x-pack/plugins/apm/public/components/alerting/rule_types/transaction_duration_rule_type/index.tsx

```diff
@@ -147,16 +153,21 @@ export function TransactionDurationRuleType(props: Props) {
   // The threshold from the form is in ms. Convert to µs.
   const thresholdMs = params.threshold * 1000;
 
-  // hide preview chart until https://github.com/elastic/kibana/pull/156625 gets merged
-  const showChartPreview = false;
-  const chartPreview = showChartPreview ? (
-    <ChartPreview
-      series={latencyChartPreview}
-      threshold={thresholdMs}
-      yTickFormat={yTickFormat}
-      uiSettings={services.uiSettings}
-    />
-  ) : null;
+  const chartPreview =
+    status === FETCH_STATUS.LOADING ? (
```

Comment:
You should also account for `FETCH_STATUS.NOT_INITIATED`:

```suggestion
    isPending(status) ? (
```